python/taichi/examples/simulation/fractal.py

```diff
@@ -25,9 +25,10 @@ def paint(t: float):
 
 def main():
     gui = ti.GUI("Julia Set", res=(n * 2, n))
-
-    for i in range(1000000):
-        paint(i * 0.03)
+    t = 0.0
+    while not gui.get_event(ti.GUI.ESCAPE, ti.GUI.EXIT):
```

Comment:
Actually maybe it's better  to check `gui.running` and set it properly?

Reply:
I found using `while not gui.get_event(ti.GUI.ESCAPE, ti.GUI.EXIT):`is a better practice because:
1.  the user can press the  `escape` key to exit
2.  Using `while gui.running` to call the gui in jupter notebook will "freeze" the window: the user cannot close the gui unless he kills the notebook thread.

Reply:
Oh @neozhaoliang that's good to know, btw can you open an issue for this and see if 
- we can fix the `freeze` when using `while gui.running`
- or update our examples to get rid of `while gui.running`
Thanks!

Reply:
A related issuse is here: #5236

Reply:
Reopened the issue so that we can properly followup :)